### PR TITLE
Revert "Copyedit c14"

### PR DESCRIPTION
**Pull request by jannes-m**
Reverts Robinlovelace/geocompr#356
I still have to do some updates, i.e., I haven't finished yet.

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Sorry if I jumped the gun. All the changes looked good. This PR seems to revert good changes.

**Comment by jannes-m** [comment]
Just to explain, I stopped working on the chapter because I needed to take care of my Katharina as she is ill. Therefore, the impression arose that I might be done with the chapter when in fact I was only taking a break, sorry!

File: 14-eco.Rmd [hunk 1]
```diff
@@ -610,12 +608,12 @@ In terms of methodology, a few additional points could be addressed:
...
-- A sequential model-based optimization (SMBO) might be preferable to the here used random search for hyperparameter optimization [@probst_hyperparameters_2018]. 
+- A sequential model-based optimization (SMBO) might be preferable to the random search for hyperparameter optimization used in this chapter [@probst_hyperparameters_2018]. 
 
 Finally, please note that random forest and other machine-learning models are frequently used in a setting with lots of observations and many predictors, much more than used in this chapter, and where it is unclear which variables and variable interactions contribute to explaining the response.
 Additionally, the relationships might be highly non-linear.
 In our use case, the relationship between response and predictors are pretty clear, there is only a slight amount of non-linearity and the number of observations and predictors is low.
-Hence, it might be worth to try a linear model.
+Hence, it might be worth it to try a linear model.
```
Comment (jannes-m) [inline 14-eco.Rmd]: @Robinlovelace Could you pls check this sentence, is it really `it might be worth it`?

**Comment by jannes-m** [comment]
Ok, finally I am done. Just one more thing, I also wrote into #334:
Figure 14.2 (p. 300) is put in between a table. Can we somehow prevent Latex from doing so. Furthermore, Figure 14.2 appears "rasterized", i.e., visually unappealing. The same is true for Figure 14.5. Note that this is not the case for the online versions of the figures. Is there a way to prevent the "rasterization" in the pdf?

**Comment by Nowosad** [comment]
> s put in between a table. Can we somehow prevent Latex from doing so. Furthermore, Figure 14.2 appears "rasterized", i.e., visually unappealing. The same is true for Figure 14.5. Note that this is not the case for the online versions of the figures. Is there a way to prevent the "rasterization" in the pdf?

@jannes-m please add those kinds of issues to https://github.com/Robinlovelace/geocompr/issues/334. We will need to work on them later.

File: 14-eco.Rmd [hunk 2]
```diff
@@ -196,7 +194,7 @@ random_points[, names(ep)] = raster::extract(ep, as(random_points, "Spatial"))
 
 Ordinations are a popular tool in vegetation science to extract the main information, frequently corresponding to ecological gradients, from large species-plot matrices mostly filled with 0s. 
 However, they are also used in remote sensing, the soil sciences, geomarketing and many other fields.
-If you are unfamiliar with ordination techniques or in need of a refresher, have a look at Michael W. Palmers [webpage](http://ordination.okstate.edu/overview.htm) for a short introduction to popular ordination techniques in ecology and at @borcard_numerical_2011 for a deeper look on how to apply these techniques in R. 
+If you are unfamiliar with ordination techniques or in need of a refresher, have a look at Michael W. Palmer' s [web page](http://ordination.okstate.edu/overview.htm) for a short introduction to popular ordination techniques in ecology and at @borcard_numerical_2011 for a deeper look on how to apply these techniques in R.
```
Comment (Nowosad) [inline 14-eco.Rmd]: Unnecessary space?

**Comment by Robinlovelace** [comment]
@jannes-m good to merge?

File: 14-eco.Rmd (hunk 1 above)
Comment (jannes-m) [reply 14-eco.Rmd]: perfect, thank, Robin!

**Pull request merged**